File: Prashant Kumar Singh/day-7/7a.cpp
class Solution {
public:
    bool isUgly(int n) {
        if (n == 0) return false;
        for (const int p : {2, 3, 5}) //p=prime
        while (n % p == 0)
        n /= p;
        return n == 1;
    }    
};       

File: Mahek Jhamb/Day 7/7A.py
class Solution:
    def isUgly(self, n: int) -> bool: 
        if n == 0: 
            return False
        number= [2,3,5]
        while n != 1:
            for x in number:
               if n % x == 0:
                   n = n // x
                   break
            else:
                return False

        return True